src/openms/include/OpenMS/CHEMISTRY/EnzymaticDigestion.h

```diff
@@ -119,6 +119,7 @@ namespace OpenMS
        @return Number of discarded digestion products (which are not matching length restrictions)
        */
     Size digestUnmodified(const StringView& sequence, std::vector<StringView>& output, Size min_length = 1, Size max_length = 0) const;
+    Size digestUnmodified(const StringView& sequence, std::vector<std::pair<Size,Size>>& output, Size min_length = 1, Size max_length = 0) const;
```

Comment:
can you describe the output somewhere ? or use a typedef ? what is a `std::vector<std::pair<Size,Size>>` supposed to be, what ca I do with this?